fbximport/src/mesh.cpp

```diff
@@ -53,7 +53,7 @@ ExportableMesh* mesh_get_geometry_data(CFbxMesh geometry)
     {
         const auto fbxVertexPositionIndex = fbxVertexPositionIndexArray[i];
         auto lVertex = lFbxPositions[fbxVertexPositionIndex];
-        auto lNormal = lFbxNormals[i];
+        auto lNormal = ignore_normals ? FbxVector4(0.0, 0.0, 0.0, 0.0) : lFbxNormals[i];
```

Comment:
Consider a "why" comment here, what does this actually "save" for us in the end? guessing its related to the _vertex_date.insert which does some check of equalness?